src/main/resources/assets/admin/common/js/notify/NotificationMessage.ts

```diff
@@ -12,10 +12,9 @@ export class NotificationMessage
         super('notification');
         this.autoHide = autoHide;
         const notificationInner = new DivEl('notification-inner');
-        const notificationRemove = new SpanEl('notification-remove');
-        notificationRemove.setHtml('X');
+        const notificationRemove = new SpanEl('notification-remove icon-close');
         this.notificationContent = new DivEl('notification-content');
-        this.notificationContent.setHtml(message);
+        this.notificationContent.setHtml(message, false);
```

Comment:
We cannot just quietly remove escaping, it was added for a reason. I suggest doing this differently: add two new properties for passing a link text and a callback function. If both these properties are set, append a link to the notification text which will execute a specified callback on click.

Reply:
Refactored addAction() method to append both text to the notification body and the click handler